test/tools/libtesteth/Options.cpp

```diff
@@ -48,7 +48,8 @@ void printHelp()
     cout << setw(35) << "-g <index>" << setw(25) << "Set the transaction gas array index when running GeneralStateTests\n";
     cout << setw(35) << "-v <index>" << setw(25) << "Set the transaction value array index when running GeneralStateTests\n";
     cout << setw(35) << "--singletest <TestName>" << "Run on a single test\n";
-    cout << setw(35) << "--singletest <TestFile> <TestName>" << "Run on a single test from file\n";
+    cout << setw(35) << "--singletest <TestFile>" << "Run tests from file. Require -t <TestSuite>\n";
```

Comment:
I would strongly prefer this to be another, separate option for pointing it to a file

Reply:
actually I met quite amount of people who always try to pass file to this optio instead of testname

Reply:
right, because there's no other option that clearly looks like it should point to the file

Reply:
+1 with @gumb0 It's really confusing that the same argument option takes very different types of values.